src/murmur/Messages.cpp

```diff
@@ -848,7 +888,33 @@ void Server::msgUserState(ServerUser *uSource, MumbleProto::UserState &msg) {
 		userEnterChannel(pDstServerUser, c, msg);
 		log(uSource, QString("Moved %1 to %2").arg(QString(*pDstServerUser), QString(*c)));
 		bBroadcast = true;
+
+		if (ChannelListener::isListening(pDstServerUser, c)) {
+			// If a user joins a channel (s)he has been listening to before, it means that this user will no
```

Comment:
Has this been tested? At least in the UI this is not reflected. I've tested it locally (be in ChanA, listen to ChanB and then join ChanB). Result:
![grafik](https://user-images.githubusercontent.com/823537/96775694-3061fa80-13e8-11eb-8430-7365157ce722.png)

Reply:
This has been changed later on. The restriction of not being allowed to listen to the channel you're currently in, has been lifted in #4129

Reply:
Oh, great :) True, I didn't check the current state of the code but the explanation in #4129 makes sense. Thanks for explaining :+1:

Reply:
Sure, no problem :)